Makefile

```diff
@@ -83,7 +84,7 @@ deploy: deploy-setup deploy-image
 deploy-example: deploy
 	oc create -n $(NAMESPACE) -f hack/cr.yaml
 
-test-e2e: deploy-image
+test-e2e:
```

Comment:
why is this removed from here? the e2e tests currently rely on an image to be produced

Reply:
It does rely on the image to be produced but CI has a mechanism for us to use the one built by CI and not one we have to both build and deploy.  This is the reason for the introduction of [1].  I think it's reasonable to require people who are manually running this test to explicitly build and deploy their own image.  IMO we need to cater to CI which is supposed to make our lives easier.
[1] https://github.com/openshift/cluster-logging-operator/pull/88/files#diff-aa5c0c7202e9c241e0c57a1d903f9ec2R21

Reply:
Ack, can we update our README to describe this then?

Reply:
That's really vague... by that I don't understand what I would set that variable to. I was meaning something to state that you will need to `make deploy-image` manually...

Reply:
@ewolinetz better? https://github.com/openshift/cluster-logging-operator/pull/88/files#diff-04c6e90faac2675aa89e2176d2eec7d8R93

Reply:
works for me!